consensus/src/block_graph.rs

```diff
@@ -90,15 +117,26 @@ impl ActiveBlock {
     }
 }
 
+/// Exportable version of ActiveBlock
+/// Fields that can be easily recomuted were left out
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct ExportActiveBlock {
+    /// The block itself, as it was created
     pub block: Block,
-    pub parents: Vec<(BlockId, u64)>, // one (hash, period) per thread ( if not genesis )
-    pub children: Vec<BlockHashMap<u64>>, // one HashMap<hash, period> per thread (blocks that need to be kept)
-    pub dependencies: BlockHashSet,       // dependencies required for validity check
+    /// one (block id, period) per thread ( if not genesis )
+    pub parents: Vec<(BlockId, u64)>,
+    /// one HashMap<Block id, period> per thread (blocks that need to be kept)
+    /// Children reference that block as a parent
+    pub children: Vec<BlockHashMap<u64>>,
+    /// dependencies required for validity check (TODO: not sure if it's used)
```

Comment:
same here

Reply:
used for graph pruning (see previous answer)